x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/ldap/support/LdapUtils.java

```diff
@@ -187,7 +198,8 @@ protected void doRun() throws Exception {
 
             @Override
             public void onFailure(Exception e) {
-                LOGGER.debug("LDAP bind [{}] failed for [{}] - [{}]", bind, ldap, e.toString());
+                Level logLevel = systemBind ? Level.WARN : Level.DEBUG;
+                LOGGER.log(logLevel, "LDAP {} bind [{}] failed for [{}] - [{}]", systemBind ? "system" : "user", bind, ldap, e.toString());
```

Comment:
```suggestion
                LOGGER.log(logLevel, "LDAP [{}] bind [{}] failed for [{}] - [{}]", systemBind ? "system" : "user", bind, ldap, e.toString());
```
?

Reply:
My view is no, but I'm open to arguments. If I'd written
 ```
    if (systemBind) {
        LOGGER.warn("LDAP system bind [{}] failed for [{}] - [{}]", bind, ldap, e.toString());
    } else {
        LOGGER.debug("LDAP user bind [{}] failed for [{}] - [{}]", bind, ldap, e.toString());
    }
```
There would be no argument that the `system` / `user` should be bracketed, but because it's been condensed into a single statement it should be? That's letting the structure of the code dictate the format of the log messages which doesn't seem right.